official/utils/logging/logger.py

```diff
@@ -19,13 +19,25 @@
 from __future__ import print_function
 
 import datetime
+import glob
 import json
+import multiprocessing
 import numbers
 import os
+import re
+
+# pylint: disable=g-bad-import-order
+# Note: cpuinfo and psutil are not installed in the TensorFlow OSS tree.
```

Comment:
Presumably this will become a requirement then of any machine or person trying to run the benchmarks. We can include that in a README when it exists; for now, might be worth noting again in the module docstring.